src/controllers/engineController.ts

```diff
@@ -0,0 +1,46 @@
+import Project from "../models/projectModel";
+
+const engineController = {
+  getEngines: async (req: any, res: any) => {
+    try {
+      const projects = await Project.find();
+  
+      let sastCount = 0;
+      let scaCount = 0;
+      let iacCount = 0;
+  
+      projects.forEach((project) => {
```

Comment:
This forEach could be reduced a lot more since you are doing the exact same thing by simply changing the project array so you can pass it to a function that takes it as a parameter.

Reply:
Fixed it, now the function gets the engines by parameters.